.github/workflows/kind.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     env:
       REGISTRY: localhost:5000
     strategy:
+      fail-fast: false
```

Comment:
Makes sense! My concern there is speed, but it's easy enough to modify the Makefile to run a subset of tests if something keeps failing consistently because of the code in the PR.